src/pages/Home/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { TweetsContainer } from '../../components/TweetsContainer';
+import { useNavigate } from 'react-router-dom';
+import { useUser } from '../../contexts/UserContext';
+
+export function Home() {
+  const { username } = useUser();
+  const navigate = useNavigate();
+
+  useEffect(function navigateAwayIfNotLoggedIn() {
+    if (!username) navigate('/');
+  }, []);
+
+  return (
+    <>
+      <TweetsContainer />
+    </>
```

Comment:
This is a bad component in my opinion, right now we are just rendering another component which is an anti-pattern

Reply:
Why would this be considered a bad programming practice? Is it because we are re-rendering a component we don't need to re-render (it is unnecessary)? Would memoization help?

Reply:
Yes, this is bad practice. It doesn't do anything just calls something else.